blurb/blurb.py

```diff
@@ -745,19 +745,18 @@ def test(*args):
 
 
 def find_editor():
-    for var in 'GIT_EDITOR', 'EDITOR':
-        editor = os.environ.get(var)
-        if editor is not None:
-            return editor
```

Comment:
I agree that git-wrapper was not a good definition, more like blurb is implemented using git.

My reasoning is:
1. blurb serve python developers
2. python developers must use git now
3. If you configure git properly blurb should work out of the box.
    if git commit uses your editor, so will blurb

This patch does not remove support for EDITOR - git var GIT_EDITOR supports it. From git manual:

       GIT_EDITOR
           Text editor for use by Git commands. ... The order of preference is the 
           $GIT_EDITOR environment variable, then core.editor configuration, then
           $VISUAL, then $EDITOR, and then the default chosen at compile time,
          which is usually vi.

I don't see a need for customizing git behavior. Delegating the editor choice to git
will make this tool easy to use for new developers, assuming they are already
using git.